integration-tests/paramchangeproposal.ts

```diff
@@ -38,32 +38,32 @@ const proposal = new ParameterChangeProposal('testing params', 'yay!', {
     SlashFractionDowntime: new Dec(213.123),
   },
   treasury: {
-    taxpolicy: new PolicyConstraints(0, 100, new Coin('unused', 0), 3),
-    rewardpolicy: new PolicyConstraints(
+    TaxPolicy: new PolicyConstraints(0, 100, new Coin('unused', 0), 3),
+    RewardPolicy: new PolicyConstraints(
       0,
       1023423340,
       new Coin('unused', 0),
       3
     ),
-    seigniorageburdentarget: new Dec('2342.234234'),
-    miningincrement: new Dec(23423423423.234234234234982),
-    windowshort: 50,
-    windowlong: 2,
-    windowprobation: 30,
+    SeigniorageBurdenTarget: new Dec('2342.234234'),
+    MiningIncrement: new Dec(23423423423.234234234234982),
+    WindowShort: 50,
+    WindowLong: 2,
+    WindowProbation: 30,
```

Comment:
I think these variable should start with lower case like `taxPolicy`. Usually the upper case used for Classes.

Reply:
No, it is been changed